### PR TITLE
CHANGELOG: update change log for 0.2.4 release

### PR DESCRIPTION
**Pull request by ffmancera**
Signed-off-by: Fernando Fernandez Mancera <ffmancera@riseup.net>

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,4 +1,14 @@
 # Changelog
+## [0.2.4] - 2020-02-12
+### Bug Fixes
+ - Reduce minimal required MAC address length to 4 octets
+ - Raise NmstateTimeoutError on mainloop timeout
+ - Validate duplicate interface names
```
Comment (tyll) [inline CHANGELOG]: IMHO this should be something like "Warn early when the same interface name is mentioned multiple times" to focus on the user-visible behavior change.

File: CHANGELOG [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+ - Support ovs-bridge and ovs-interface with the same name
+
+### New features
+ - Support link-aggregation port in OVS
```
Comment (cathay4t) [inline CHANGELOG]: We only have NM implementation, not public interface support.

File: CHANGELOG [hunk 3]
```diff
@@ -1,4 +1,14 @@
 # Changelog
+## [0.2.4] - 2020-02-12
+### Bug Fixes
+ - Reduce minimal required MAC address length to 4 octets
+ - Raise NmstateTimeoutError on mainloop timeout
+ - Validate duplicate interface names
+ - Support ovs-bridge and ovs-interface with the same name
```
Comment (tyll) [inline CHANGELOG]: This is not supported, we only have a test that shows that it is not supported. This line can be dropped IMHO.

File: CHANGELOG (hunk 2 above)
Comment (tyll) [inline CHANGELOG]: This is not finished, yet, only the internals are done.

File: CHANGELOG [hunk 4]
```diff
@@ -1,4 +1,14 @@
 # Changelog
+## [0.2.4] - 2020-02-12
+### Bug Fixes
+ - Reduce minimal required MAC address length to 4 octets
+ - Raise NmstateTimeoutError on mainloop timeout
```
Comment (tyll) [inline CHANGELOG]: This is more a feature than a bugfix, since the exception is new.

Comment (ffmancera) [reply CHANGELOG]: Done, thanks!

File: CHANGELOG (hunk 2 above)
Comment (ffmancera) [reply CHANGELOG]: Thanks, removed.

File: CHANGELOG (hunk 3 above)
Comment (ffmancera) [reply CHANGELOG]: Done, thanks!

File: CHANGELOG (hunk 2 above)
Comment (ffmancera) [reply CHANGELOG]: Done, thanks!

File: CHANGELOG (hunk 1 above)
Comment (ffmancera) [reply CHANGELOG]: Done, thanks!

**Pull request merged**